WalletWasabi.Documentation/Guides/MacOsSigning.md

```diff
@@ -0,0 +1,45 @@
+# MacOS release environment guide (for signing and notarizating)
+
+## How to create appleId and get code signing certificate
+
+1. Get a macintosh machine. Apple developer program require two factor authentication.
+2. Enroll in the apple developer program.
+3. Create a New Certificate https://developer.apple.com/account/resources/certificates/list type: Developer ID Application (This certificate is used to code sign your app for distribution outside of the Mac App Store.)
+4. Download the .cer file.
+5. Use Keychain Access to export Personal Information Exchange (.p12) file. https://knowledge.digicert.com/solution/SO25463.html
+6. Download the .p12 file to desktop and run WalletWasabi.Packager/Content/Osx/addcert.sh => Go to terminal ./addcert.sh - might run chmod a+rx addcert.sh.
```

Comment:
I think "Download" is not the word.

Reply:
I am not a macOS guy but i think you need `sudo chmod u+x addcert.sh` first, am I right?

Reply:
Fixed.